### PR TITLE
drop numpy 1.6 support

### PR DESCRIPTION
**Pull request by hamogu**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -2,7 +2,10 @@
 0.3 (unreleased)
 ----------------
 
-- No changes yet
+General
+^^^^^^^
+
+- Drop numpy 1.6 support, minimal required version is not numpy 1.7 [#327]
```
Comment (astrofrog) [inline CHANGES.rst]: not -> now

**Comment by larrybradley** [comment]
Looks good except for the one typo in the changelog.  Thanks.

**Comment by hamogu** [comment]
fixed typo -> can be merged.

**Comment by eteq** [comment]
Looks good - merging!

**Pull request merged**